AssetRipper.TextureDecoder/Rgb/Formats/ColorRGB32Half.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace AssetRipper.TextureDecoder.Rgb.Formats
+{
+    /// <summary>
+    /// Also called R11G11B10_FLOAT
+    /// </summary>
+    public struct ColorRGB32Half : IColor<Half>
+    {
+        private uint bits;
+
+        [MethodImpl(OptimizationConstants.AggressiveInliningAndOptimization)]
+        private static Half ToHalf(ushort value)
+        {
+        #if NET5_0_OR_GREATER
+            return Unsafe.As<ushort, Half>(ref value);
+        #else
+            return ToHalf(value);
+        #endif
+        }
+
+        [MethodImpl(OptimizationConstants.AggressiveInliningAndOptimization)]
+        private static uint FromHalf(Half value)
+        {
+        #if NET5_0_OR_GREATER
+            return Unsafe.As<Half, ushort>(ref value);
+        #else
+            return FromHalf(value);
+        #endif
+        }
```

Comment:
Always use `Unsafe.As` for converting `Half` bits to a `Half` value, and vice versa.